src/components/screen-view/ScreenWithLargeHeader.tsx

```diff
@@ -52,6 +52,7 @@ export const ScreenWithLargeHeader = ({
             {headerChildren && (
               <View
                 ref={!title ? focusRef : undefined}
+                accessible={!title}
```

Comment:
A little unsure about if it is correct with `accessible: false` when `title` is present 🤔 But at least it is not crashing.